### PR TITLE
:lipstick: :art: Created booklets-page hero section

### PR DESCRIPTION
**Pull request by Sarah-Wambui**
# Description

Created booklets-page her section

#73 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by paulkahura** [comment]
kindly include a screenshot for both mobile and desktop for review

File: src/css/style.css
Comment (paulkahura) [inline src/css/style.css:1]: Kindly include the screenshot for both mobile and desktop

**Review by paulkahura — CHANGES_REQUESTED** [review]

**Comment by Sarah-Wambui** [comment]
Desktop
mobile

**Comment by JAPHETHNYARANGA** [comment]
Good attempt closing this pr as it is already done, please pick another issue

**Pull request closed without merging**